modules/languages/php.nix

```diff
@@ -0,0 +1,75 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib;
+with builtins; let
+  cfg = config.vim.languages.php;
+in {
+  options.vim.languages.php = {
+    enable = mkEnableOption "PHP language support";
+
+    treesitter = {
+      enable = mkEnableOption "Enable PHP treesitter" // {default = config.vim.languages.enableTreesitter;};
+      package = nvim.types.mkGrammarOption pkgs "php";
+    };
+
+    lsp = {
+      enable = mkEnableOption "PHP LSP support" // {default = config.vim.languages.enableLSP;};
+
+      package = mkOption {
+        description = "lsp pacakge";
+        type = types.package;
+        default = pkgs.phpactor;
+      };
+    };
+
+    analyzer = {
+      package = mkOption {
+        description = "analizer pacakge";
+        type = types.package;
+        default = pkgs.php81Packages.phan;
+      };
+    };
```

Comment:
if I'm understanding the diff correctly, the analyzer setup is gated between `lsp.enable` thus the analyzer package should also be under the LSP attribute set.